Documentation/observability/visibility.rst

```diff
@@ -62,6 +62,8 @@ endpoint of that pod, for example:
     NAME                       ENDPOINT ID   IDENTITY ID   INGRESS ENFORCEMENT   EGRESS ENFORCEMENT   VISIBILITY POLICY   ENDPOINT STATE   IPV4           IPV6
     coredns-7d7f5b7685-wvzwb   1959          104           false                 false                OK                  ready            10.16.75.193   f00d::a10:0:0:2c7
 
+Cilium run should enable Cilium Endpoint Status set to policy (``--endpoint-status=policy``) to make INGRESS ENFORCEMENT, EGRESS ENFORCEMENT, VISIBILITY POLICY fields visible.
+
```

Comment:
```suggestion
In order for Cilium to populate the ``INGRESS ENFORCEMENT``, ``EGRESS ENFORCEMENT``, and ``VISIBILITY POLICY`` fields, it must run with ``--endpoint-status=policy`` to make field values visible.
```

Reply:
updated :)

Reply:
Could you wrap the text like other paragraphs?

Reply:
do you mean to put it in the code block or smth else?

Reply:
I mean that instead of being one long line, it's wrapped into smaller lines, like other paragraphs in the file.